lib/org/chromium/webidl/ParserRunner.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'ParserRunner',
+  extends: 'foam.box.Runnable',
+
+  requires: ['org.chromium.webidl.PipelineMessage'],
+
+  documentation: 'Runnable Box that parsers IDLFileContents into ASTs.',
+
+  methods: [
+    function run(message) {
+      if (!this.PipelineMessage.isInstance(message))
+        return this.error(new Error("ParserRunner: Run() expects a" +
```

Comment:
nit: Here and elsewhere: Let's use a template string or single-quoted. Reserve double-quoted for when template string is not appropriate and the string contains single quotes.